app/bundles/PageBundle/Controller/PageController.php

```diff
@@ -113,8 +113,14 @@ public function indexAction($page = 1)
             $filter['force'][] = ['column' => 'p.translationParent', 'expr' => 'isNull'];
         }
 
-        $orderBy    = $this->get('session')->get('mautic.page.orderby', 'p.title');
-        $orderByDir = $this->get('session')->get('mautic.page.orderbydir', 'DESC');
+        $orderBy    = $this->get('session')->get('mautic.page.orderby', 'p.dateModified');
+
+        $orderByDir = $this->get('session')->get('mautic.page.orderbydir');
```

Comment:
We can replace it with one line:

`$orderByDir = $this->get('session')->get('mautic.page.orderbydir', 'DESC')`